Description: Fix FTBFS due to missing GdkSpanFunc definition
 .
 The sequence of #include directives is important - if gtk/gtkglwidget.h
 is included first, before widget.h, it includes gdk.h without undefing
 GDK_DISABLE_DEPRECATED - this leads to GdkSpanFunc not being defined
 and compilation errors. 
 The fix is to first include widget.h which includes the right gdkmm header
 which defines GDK_DISABLE_DEPRECATED before including gdk.h.
Forwarded: no
Bug-Ubuntu: http://launchpad.net/bugs/662572

Index: gtkglextmm-1.2.0/gtkglext/gtkmm/gl/widget.cc
===================================================================
--- gtkglext/gtkmm/gl/widget.cc	2010-12-08 13:31:59.651270000 +0100
+++ gtkglext/gtkmm/gl/widget.cc	2010-12-08 13:32:10.871270000 +0100
@@ -17,10 +17,10 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307  USA.
  */
 
-#include <gtk/gtkglwidget.h>
-
 #include "widget.h"
 
+#include <gtk/gtkglwidget.h>
+
 namespace Gtk
 {
   namespace GL
